### DISCUSSION TITLE
Where is the documentation?

### DISCUSSION THREAD

**Discussion by xegulon** [fnando/i18n-js#637] [Q&A]
Hi, I'm having a hard time finding the docs for this wonderful lib.
Is it the same as : http://i18njs.com/ ?
Thanks a lot.

**Comment by PikachuEXE** [comment]
1. http://i18njs.com/ **is not** a page for this project
   It seems linked to https://www.npmjs.com/package/roddeh-i18n
2. If you are looking for 3.x doc visit https://github.com/fnando/i18n-js/tree/v3
  The default branch points to v4 (unreleased)
